### ISSUE TITLE
Achieve Decision: Do we need to validate signatures from staging in parallel?

### ISSUE THREAD

**Issue by LappleApple** [kubernetes-sigs/promo-tools#1127]
## Objective 
- SIG reaches a decision based on data that shows what's realistic.
- In the future, SIG uses this decision to plan future work.

## Tasks to achieve the objective
By 1-3 contributors:
- [ ] Do research on the topic
- [ ] Answer the question, "Maybe a reconciliation job that handles this as a regular CI task would work?" and any others raised by the group that are related to this goal
     - [ ] Collect additional questions as part of your research phase
- [ ] Prepare a brief (1-2 pages max) proposal evaluating pros, cons, and tradeoffs
- [ ] Share proposal with SIG (mailing list, Slack thread, at community meeting)
By the SIG/group:
- [ ] SIG members provide input on findings/raise questions
- [ ] After a timeboxed review period (to be determined), SIG members regroup to make a decision
- [ ] Log decision for community awareness
- [ ] Plan next steps

## Context and things to think about while working on this task
- This relates to potential work to make the image promoter less monolithic
- We currently also sign things in parallel

**Comment by saschagrunert** [comment]
Closing in favor of https://github.com/kubernetes-sigs/promo-tools/issues/1701, thank you for all the work which went into this. It's consolidated into the single issue.
